bin/build_integrations_tests_runner

```diff
@@ -2,14 +2,16 @@
 
 function _build_test_integration_exec() {
   echo "Dropping into virtual environment..."
-  python3 -m venv venv
+  python -m venv venv
```

Comment:
i still think we should have `python3` even though we define python 3.9

Reply:
I'm using pyenv and you set the Pyhton to the desired version, you can simply call python